crates/sui-verifier-transactional-tests/tests/init/must_have_txn_context.mvir

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# publish
+module 0x0.M {
+    import 0x2.TxContext;
+    init() {
+        label l0:
+        abort 0;
+    }
+}
+
+
+//# publish
+module 0x0.M {
+    import 0x2.TxContext;
+    init(ctx: &mut TxContext.TxContext, ctx2: &mut TxContext.TxContext) {
```

Comment:
Is there a test where there is a `TxContext` earlier and some "normal" arguments later in the list?